docs/installation.md

```diff
@@ -40,6 +40,8 @@ If you want to enable Google Maps on iOS, obtain the Google API key and edit you
 
 (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
+// The plus sign '+' below is part of the statement. Do NOT remove it. 
+// Replace _YOUR_API_KEY_ below with your projcet's Google API Key
 +  [GMSServices provideAPIKey:@"_YOUR_API_KEY_"]; // add this line using the api key obtained from Google Console
```

Comment:
This is a standard diff view. It shouldn't be changed :)

Reply:
What I am trying to say is that in this case, the "+" sign is NOT part of the standard diff view. It is actually part of the code, and it should be typed as it is!

Reply:
We are not trying to get the consumer to type the `+` as it is. While it might build while leaving the `+` in, the code should look like this:
```objc
- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
  [GMSServices provideAPIKey:@"some-api-key"];
  self.moduleName = @"MapsTest";
  // You can add your custom initial props in the dictionary below.
  // They will be passed down to the ViewController used by React Native.
  self.initialProps = @{};

  return [super application:application didFinishLaunchingWithOptions:launchOptions];
}
```

not like this:
```objc
- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
  + [GMSServices provideAPIKey:@"some-api-key"];
  self.moduleName = @"MapsTest";
  // You can add your custom initial props in the dictionary below.
  // They will be passed down to the ViewController used by React Native.
  self.initialProps = @{};

  return [super application:application didFinishLaunchingWithOptions:launchOptions];
}
```

Any reason why you would want to leave the `+` in there?

Reply:
You are right. Sorry.